eiskaltdcpp-qt/src/FinishedTransfers.h

```diff
@@ -387,7 +387,7 @@ friend class dcpp::Singleton< FinishedTransfers<isUpload> >;
     }
 
     void openFile(QString file){
-        if (!file.startsWith("/"))
+        if (!file.startsWith(QString("/")))
```

Comment:
Why not QChar? It should work a bit faster.

Reply:
It not work with QChar
eiskaltdcpp-qt/src/FinishedTransfers.h:508:34: error: no matching function for call to 'QChar(const char [2])'
                 if (f.startsWith(QChar("/")))
                                  ^~~~~~~~~~

Reply:
QChar("/") -> QChar('/');

Reply:
or f.startsWith('/')

Reply:
OK. It work with QChar('/'). Pull request updated.

Reply:
@pavel-pimenov It not work with startsWith('/')

Reply:
> or f.startsWith('/')

It will not work due to ambiguous which constructor to use and which function to call.